nucleus/distributions/nucleus-common/src/main/resources/bin/nadmin.java

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
These three "java       scripts" are nearly copy pasted, but they have no dependencies and are compiled on execution. It has advantages and disadvantages.

```
- Needs more memory needed than with bash/bat
- Slower than bash/bat 
  - controversial as it doesn't parse asenv and config.sh
  + jar files executed by that are much heavier problem, seconds vs milliseconds
+ OS independent
```